ext/node/polyfills/internal/crypto/diffiehellman.ts

```diff
@@ -219,10 +225,21 @@ export class DiffieHellmanGroup {
 }
 
 export class ECDH {
+  curve: EllipticCurve; // the selected curve
+  privbuf: Buffer; // the private key
+  pubbuf: Buffer; // the public key
+  ephemeral_secret_resource: number; // the resource id in the table
```

Comment:
I believe these should be maybe private fields (ie. prefixed with `#`

Reply:
684ad96e38e8afec2ba4fef676531295fe906555